<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="player_" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="open_btn_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pause_btn_">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="stop_btn_">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
